ixje/neo-python
# remove logzero dependancies

**What current issue(s) does this address, or what feature is it adding?**
- addresses https://github.com/CityOfZion/neo-python/pull/934#issuecomment-496112217

- supports https://github.com/CityOfZion/neo-python/pull/934

**How did you solve this problem?**
- searched for logzero imports
- replaced with `log_manager` where necessary: Transaction.py & Send.py
- completed manual testing to verify expected result

**How did you make sure your solution works?**
manual testing

**NOTE:** I don't think I can add individual unittests for this issue because running `make test` displays every level of `logger`.

**Are there any special changes in the code that we should be aware of?**
no

**Please check the following, if applicable:**

- [ ] Did you add any tests?
- [x] Did you run `make lint`?
- [x] Did you run `make test`?
